src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/MethodConfigurationComponent.scala

```diff
@@ -140,6 +142,28 @@ trait MethodConfigurationComponent {
       }
     }
 
+    object DeleteMethodConfiguration extends RawSqlQuery {
+      val driver: JdbcDriver = MethodConfigurationComponent.this.driver
+
+      def deleteAction(workspaceId: UUID) = {
+        val tables: Seq[SQLActionBuilder] = Seq(sql"METHOD_CONFIG_INPUT", sql"METHOD_CONFIG_OUTPUT",
+          sql"METHOD_CONFIG_PREREQ")
+
+        def baseQuery(table: SQLActionBuilder) = {
+          concatSqlActions(sql"delete t from ", table,
+            sql""" as t inner join METHOD_CONFIG as mc on mc.id=t.method_config_id
+                 where mc.workspace_id=${workspaceId}""")
+        }
+
+        DBIO.sequence(tables map { table =>
+          baseQuery(table).as[Int]
+        }) andThen {
+          sqlu"delete mc from METHOD_CONFIG mc where workspace_id=${workspaceId}"
```

Comment:
This doesn't need to be raw SQL.  Remove this and update `WorkspaceComponent.deleteWorkspaceSubmissions` to match `WorkspaceComponent.deleteWorkspaceEntitiesAndAttributes`